[2020-06-17 11:11:47,542] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: email_example.hello_task 2020-06-17T11:08:00+00:00 [queued]>
[2020-06-17 11:11:47,552] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: email_example.hello_task 2020-06-17T11:08:00+00:00 [queued]>
[2020-06-17 11:11:47,553] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 11:11:47,553] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-17 11:11:47,553] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 11:11:47,561] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): hello_task> on 2020-06-17T11:08:00+00:00
[2020-06-17 11:11:47,563] {standard_task_runner.py:53} INFO - Started process 4355 to run task
[2020-06-17 11:11:47,619] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: email_example.hello_task 2020-06-17T11:08:00+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-17 11:11:47,632] {python_operator.py:114} INFO - Done. Returned value was: Hello world!
[2020-06-17 11:11:47,643] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=email_example, task_id=hello_task, execution_date=20200617T110800, start_date=20200617T111147, end_date=20200617T111147
[2020-06-17 11:11:57,535] {logging_mixin.py:112} INFO - [2020-06-17 11:11:57,534] {local_task_job.py:103} INFO - Task exited with return code 0
